README.md

```diff
@@ -2,3 +2,5 @@
 M+ bot for Silverash
 
 Leaderboard for M+ IO scores in guild 
+
+Testing Github Desktop XD
```

Comment:
Re-requesting changes so you can test that too

Also can't you add an actual description lmao

Reply:
I hope your test worked pog

Reply:
Haha it worked! Thanks friends!